src/Application/ApproverRepository.php

```diff
@@ -16,6 +16,11 @@ public function getApproverCategories( int $userId ): array;
 	 */
 	public function getApproversWithCategories(): array;
 
+	/**
+	 * @return string[]
+	 */
+	public function getAllAssignedCategories(): array;
```

Comment:
`getAllCategoriesWithApprovers` seems a bit better as it avoids `assigned` (a new concept, or at least a new name for one)